addons/binding/org.openhab.binding.hdpowerview/META-INF/MANIFEST.MF

```diff
@@ -13,8 +14,8 @@ Export-Package:
  org.openhab.binding.hdpowerview.handler
 Import-Package: 
  com.google.gson,
- javax.ws.rs.client;version="2.0.0",
- javax.ws.rs.core;version="2.0.1",
+ javax.ws.rs.client,

```

Comment:
@beowulfe ae you aware of anything that will break because of this change?

Reply:
No, that looks safe to me.